modules/raycast/raycast_occlusion_cull.cpp

```diff
@@ -578,7 +578,17 @@ Vector2 RaycastOcclusionCull::_jitter_half_extents(const Vector2 &p_half_extents
 	// False shown can lower percentage that are occluded, and therefore performance.
 	jitter *= 0.66f;
 
-	return p_half_extents + jitter;
+	return jitter;
+}
+
+Rect2 RaycastOcclusionCull::_get_viewport_rect(const Projection &p_cam_projection) const {
```

Comment:
```suggestion
Rect2 _get_viewport_rect(const Projection &p_cam_projection) {
```
I believe you can make it a free function in the .cpp. It's not that common across godot's codebase but given it has only one local use + it might be made obsolete by the other PR, it's probably a valid exception.